ansible/roles/zabbix_es/templates/ESzabbix.py.j2

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
```

Comment:
yea, all ansible galaxy does is download the source and extract it to `roles_path`